governance/votes/Executive vote - September 17, 2021.md

```diff
@@ -0,0 +1,116 @@
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://community-development.makerdao.com/en/learn/governance/how-voting-works/) to learn how voting works, and this [wallet setup guide](https://community-development.makerdao.com/en/learn/governance/voting-setup/) to set up your wallet to vote.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes** will occur within the Maker Protocol:
+- The GUNIV2DAIUSDC1-A vault type will be onboarded to the Maker Protocol with the parameters detailed below.
+- The process for offboarding the KNC-A vault type will begin as detailed below.
+- Debt Ceiling Instant Access Module parameter changes will be made for PSM-PAX-A and PSM-USDC-A as detailed below.
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
+
+Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://community-development.makerdao.com/en/learn/governance/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
+
+ This executive proposal includes an office-hours modifier that means that it **can only be executed between 14:00 and 21:00 UTC, Monday - Friday**.
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+---
+
+## Proposal Details
+
+### GUNIV2DAIUSDC1-A Onboarding
+
+Due to the outcome of this [governance poll](https://vote.makerdao.com/polling/QmZ5TESw?network=mainnet#poll-detail) the GUNIV2DAIUSDC1-A vault will be created with the parameters below if this executive proposal passes.
+
+**Collateral Parameters**
+
+* Underlying Collateral: [GUNI-DAI/USDC](https://forum.makerdao.com/t/g-uni-dai-usdc-uniswap-v3-lp-erc20-collateral-onboarding-application/9026)
+* [Stability Fee](https://community-development.makerdao.com/en/learn/governance/param-stability-fee): 1%
+* [Liquidation Ratio](https://community-development.makerdao.com/en/learn/governance/param-liquidation-ratio): 105%
+* [Debt Ceiling (`line`)](https://community-development.makerdao.com/en/learn/governance/param-debt-ceiling): 10 million DAI
+* [Debt Floor (`dust`)](https://community-development.makerdao.com/en/learn/governance/param-debt-floor): 10,000 DAI
+* [Target Available Debt (`gap`)](https://makerdao.world/en/learn/governance/module-dciam): 10 million DAI
+* [Ceiling Increase Cooldown (`ttl`)](https://makerdao.world/en/learn/governance/module-dciam): 8 hours
+
+
+**Liquidation Parameters**
+
+*Note that liquidations are **disabled** for this collateral type. The following parameters will only become relevant if liquidations are later enabled by Maker Governance.*
+
+-- Core --
```

Comment:
Should these be bolded? not sure how this formatting will appear on the website

Reply:
Wanted them to be less of a division, given that they are all parameters that belong in this section. We'll see how it looks in practice.